addons/kotsadm/alpha/install.sh

```diff
@@ -26,6 +26,8 @@ function kotsadm() {
     if [ -n "$PROXY_ADDRESS" ]; then
         render_yaml_file "$DIR/addons/kotsadm/alpha/tmpl-kotsadm-proxy.yaml" > "$DIR/kustomize/kotsadm/kotsadm-proxy.yaml"
         insert_patches_strategic_merge "$DIR/kustomize/kotsadm/kustomization.yaml" kotsadm-proxy.yaml
+        render_yaml_file "$DIR/addons/kotsadm/alpha/tmpl-kotsadm-api-proxy.yaml" > "$DIR/kustomize/kotsadm/kotsadm-api-proxy.yaml"
```

Comment:
The GraphQL API makes an outbound request to test registry credentials so this deployment needed proxy env vars also.